x/perpetual/module.go

```diff
@@ -117,7 +117,11 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
 	m := migrations.NewMigrator(am.keeper)
-	err := cfg.RegisterMigration(types.ModuleName, 10, m.V11Migration)
+	err := cfg.RegisterMigration(types.ModuleName, 9, m.V10Migration)
+	if err != nil {
+		panic(err)
+	}
+	err = cfg.RegisterMigration(types.ModuleName, 10, m.V11Migration)
```

Comment:
@cryptokage1996 can we use one version increment (version 9 to 10) instead of two (9 to 10 and then 10 to 11) for this module as part of the new upgrade?